api/spec/spec.go

```diff
@@ -0,0 +1,234 @@
+package spec
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+// AssertHTTPAPISpec runs a test suite of HTTP requests against the given base URL
+// to assert it conforms to the qri core API specification. Spec is defined in
+// the "open_api_3.yaml" file in the api package
+func AssertHTTPAPISpec(t *testing.T, baseURL, specPackagePath string) {
+	t.Helper()
+
+	// Create a mock data server. Can't move this into the testRunner, because we need to
+	// ensure only this test is using the server's port "55555".
```

Comment:
`newMockDataServer` and `api/spec/testdata/dataset.json` both use 55556, which disagree with the comment. I think the comment needs to be updated?